pkgs/by-name/on/onedrive/package.nix

```
@@ -26,6 +27,15 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-Lek1tW0alQQvlOHpz//M/y4iJY3PWRkcmXGLwjCLozk=";
   };
 
+  patches = [
+    # remove when updating to v2.5.4
+    (fetchpatch {
+      name = "fix-openssl-version-check-error.patch";
+      url = "https://patch-diff.githubusercontent.com/raw/abraunegg/onedrive/pull/2981.patch";
+      hash = "sha256-IMXCKYvdpk9Drb/bNDlVQl/hUhi1It0jQpFbT3qKzm0=";
```

```suggestion
      url = "https://github.com/abraunegg/onedrive/commit/d956318b184dc119d65d7a230154df4097171a6d.patch";
      hash = "";
```

https://github.com/abraunegg/onedrive/commit/d956318b184dc119d65d7a230154df4097171a6d

Patch generated from PR may not be stable.